Dockstore.cwl

```diff
@@ -135,6 +135,7 @@ doc: |
         "path":"bams/normal_sample.bam",
         "class":"File"
       },
+      "download_reference_files": "false",
```

Comment:
Can we get this change into https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockstore_cwl.json as well?

Reply:
added